conbench/entities/commit.py

```diff
@@ -415,25 +414,40 @@ def session(self):
             session.headers = {"Authorization": f"Bearer {token}"}
         return session
 
-    def test_commit(self, sha):
+    def _mocked_get_response(self, sha) -> dict:
```

Comment:
I did this quick rename from `test_commit()` to `_mocked_get_response()` to hopefully make the name a tiny bit better reflect the purpose of this method.